README.md

```diff
@@ -1,174 +1,166 @@
+| no-sign-request | - | Do not sign requests. Credentials will not be loaded if this argument is provided |
+| operation | string | Operation type: `put`, `multipartput`, `get`, `puttagging`, `updatemeta`, `randget`, `delete`, `options`, `head`, `restore`. Default: `put`
+| overwrite | int | Turns a PUT/GET/HEAD into an operation on the same S3 key. `1`: all writes/reads are to same object, `2`: threads clobber each other but each write/read is to unique objects | 
+| partsize | int | Size of each part in bytes. Only has an effect when a multipart put is used. Min: `5242880` (5MiB). Default: `5242880` (5MiB)|
+| prefix | string | Object name prefix. Default: `testobject` |
+| profile | string | Use a specific profile from [AWS CLI credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) |
+| range | string | Specify range header for GET requests |
+| ratelimit | float | The total number of operations per second across all threads. Default: `1.7976931348623157e+308` | 
+| region | string | Region to send requests to. Default: `us-east-1` |
+| repeat | int | Repeat each S3 operation this many times: Default: `0` (do not repeat) |
+| requests | int | Total number of requests. Mutually exclusive with `duration`. Default: `1000` |
+| retrysleep | int | How long to sleep in between each retry in milliseconds. Default: `0` (exponential backoff) |
+| rr | - | Reduced redundancy storage for PUT requests | 
+| size | int | Object size in bytes. Note that s3tester is not ideal for very large objects as the entire body must be read for v4 signing and the AWS SDK does not support v4 chunked. Performance may degrade as size increases due to the use of v4 signing without chunked support. Default: `30720` |
+| tagging | string | The tag-set for the object. The tag-set must be formatted as such: `'tag1=value1&tage2=value2'`. Used for `put`, `puttagging`, `putget` and `putget9010r` |
+| tier | string | The retrieval option for restoring an object. One of `expedited`, `standard`, or `bulk`. AWS default option is standard if not specified. Default: `standard` |
+| uniformDist | string | Generates a uniform distribution of object sizes given a min-max size. Allowed values: `10` to `20`) |
+| verify | int | Verify the retrieved data on a get operation. `0`: disable verify (default); `1`: normal put data, `2`: multipart put data. If verify equals `2`, partsize is required (default `partsize` is `5242880` bytes) |
+| workload | string | Filepath to JSON either a replay file generated by the auditAnalysis tool to play an exact workload on a grid or a `mixedWorkload` JSON file which allows a user to specify a mixture of operations. A sample mixed workload file must be in the format provided below the table |
+
+#### mixedWorkload JSON Sample File
+
+```raw
+{
+	"mixedWorkload": [{
+		"operation": "put",
```

Comment:
Replace with "operationType"

Reply:
I am not sure if this is now correct (I asked in https://github.com/s3tester/s3tester/issues/24#issuecomment-764443592 to confirm) - if it's still wrong I'll fix it later. Or feel free to modify my commit if you just want to close it faster.